tests/unit/utils/test_json.py

```diff
@@ -42,5 +45,72 @@ def test_loading_json_from_url():
         assert isinstance(document.get("columns"), list)
 
 
+def test_load_local_when_http_request_fails(dummy_mapped_url):
```

Comment:
I think this functionality should really be in a test file associated with the module we're really testing (in this case it's currently `cache.py`).